lib/rom/sql/function.rb

```diff
@@ -60,6 +60,13 @@ def qualified(table_alias = nil)
         )
       end
 
+      # @see Attribute#qualified?
+      #
+      # @api private
+      def qualified?(table_alias = nil)
+        meta[:func].args.all?(&:qualified?)
```

Comment:
@GustavoCaso is `meta[:func]` guaranteed to always be defined here? I think we should be defensive here and check that it exists.

Reply:
Good point. It's not guaranteed, because a new function object is treated as a function builder, whatever method you call on it, will result in setting `meta[:func]`.